### DISCUSSION TITLE
Transitioning Users From NZBGet to NZBGet-ng

### DISCUSSION THREAD

**Discussion by tdworz** [nzbgetcom/nzbget#131] [General]
Thanks for continuing work on NZBGet!

I'd like to start a discussion about efforts that can be made to help transition users to this fork. I personally rely on a packaged copy of NZBGet and I'd like my Linux distribution to continue to package NZBGet as it is developed in this repo. I think it would also be good to try to save people the hour I just spent researching what is going on with NZBGet as they realize the project is ~~abandoned~~ archived.

First, has any effort been made to reach out to @hugbug about including some sort of notice in the About section or `README.md` file of the [nzbget/nzbget repository](https://github.com/nzbget/nzbget) that this project exists? Having such a note is not uncommon and may save people a lot of time researching the situation. It can also help coalesce efforts around this project.

Beyond that, perhaps even asking for control over the [NZBGet organization](https://github.com/nzbget) and [nzbget.net domain](https://nzbget.net/) is worth an effort if it has not already been tried. If @hugbug would agree to giving this project the domain, perhaps it would be worth asking for the forum database as well to preserve past discussions.

Second, has any effort been made to contact downstream packagers of NZBGet to encourage them to continue their work with this code base and the future releases published here? NZBGet is distributed by [several downstream repositories](https://pkgs.org/download/nzbget).

Third, perhaps some kind of post or notice summarizing the situation could be useful, specifically in hopes that it returns prominently in search engine results. A respectful and accurate account of why this fork and project exist and plans for the future would be nice to reference [in places discussing the situation](https://www.reddit.com/r/nzbget/comments/10vjahi/nzbget_abandoned_by_developer/). I know there is a [notice on top of this repo's README.md](https://github.com/nzbget-ng/nzbget/commit/4af1bd7347213c74b0f440c2368e69d47b38c64a#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R1), but perhaps a page on the [NZBGet-ng](https://nzbget-ng.github.io/) website would be better.

I'd be willing to help with any of these efforts.

**Comment by faethon** [comment]
There might be quite some users that are using the Docker image from linuxserver.io, which used to be the most downloaded NZBGet docker image. It may be usefull to replicate the image, based on the NZBget-ng code to easily transition end users that are relying on the docker image from linuxserver. I noticed there are already quite some forks. Might be handy to have a docker image that is linked on this repo, and maintained from NZBGet-ng as well.

**Comment by beertje44** [comment]
ATM the situation is rather confusing, that is for users of the linux-server nzbget docker image. Which is probably the way to go these days if you want to run nzbget on Linux. I do however want to say that this initiative is a really pleasant surprise for me personally. It's just that there are mutiple forks for the linux server docker setup right now, it seems people with the right skills are fixing this on their own. So for the rest of us it is a bit confusing which fork to choose and how to get nzbget-ng up&running on docker/podman. Personally I've chosen https://github.com/bradpeczka/docker-nzbget-ng and managed to get it up&running by adding libcap-devel to the build step and libcap to the final image. Not that hard. Also I pinned the version to tag v21.4-rc1, but that is not what is mentioned inside nzbget, guess this is still work in progress? Apart from this I have some modification that allow me to run beets on music files and filebot on movies, but that is just for my setup and probably of little interest to the avarage user.

**Comment by ovizii** [comment]
I'm also interested how to transition from the linuxserver.io package to this new fork.

**Comment by beertje44** [comment]
Here you go: https://hub.docker.com/repository/docker/jb044/nzbget-ng/general

Still based on alpine 3.18, 3.19 doesn't seem to like pip for installing python deps. Also I use beets and filebot in my own nzbget scripts. So I added openjdk and python beets. If you don't need that just don't use it ;) It'll only get used if you script it.
